tests/integration_tests/performance/test_rate_limiter.py

```diff
@@ -152,12 +152,12 @@ def _check_tx_rate_limiting(test_microvm):
 
     # Sanity check that bandwidth with no rate limiting is at least double
     # than the one expected when rate limiting is in place.
-    assert _diff(rate_no_limit_kbps, expected_kbps) > 100
+    assert _relative_change(rate_no_limit_kbps, expected_kbps) > 1.0
```

Comment:
shall we squash this name change with the former one, please?

Reply:
no, I mean you changed this to `_diff` in some earlier commit and then changed it to `_relative_change` in a latter one. Just do the change once.